routes/web.php

```diff
@@ -19,6 +19,10 @@
     Route::get('faset', function () {
         return view('faset');
     });
+
+    Route::get('faset/admin', function () {
```

Comment:
Not opposed, but it depends on how we structure roles. I'll try to work on fleshing that out this weekend.